src/canister/user_index/src/api/user_record/get_requester_principals_canister_id_create_if_not_exists_and_optionally_allow_referrer.rs

```diff
@@ -152,11 +153,15 @@ async fn new_user_signup(user_id: Principal) -> Result<Principal, String> {
     let individual_user_canister_subnet_threshold = get_individual_user_canister_subnet_threshold();
     let individual_user_canister_subnet_batch_size =
         get_individual_user_canister_subnet_batch_size();
-    if individual_user_canisters_cnt < individual_user_canister_subnet_threshold {
+    if available_individual_user_canisters_cnt < individual_user_canister_subnet_threshold {
         let new_canister_cnt =
             individual_user_canister_subnet_batch_size.min(backup_individual_user_canister_cnt);
+
+        let max_canister_cnt: u64 =
```

Comment:
looks like max_canister_cnt value is same as new_canister_cnt . Not expecting this to be changed.